src/components/custom-fonts.js

```diff
@@ -8,22 +8,22 @@ import React, {Component} from 'react';
 import {styled} from 'fusion-plugin-styletron-react';
 import {withFontLoading as FontHOC} from 'fusion-plugin-font-loader-react';
 
-// FIXME: as of React 16, <div> props are written to html (e.g. fontstyles)
+// FIXME: as of React 16, <div> props are written to html (e.g. $fontstyles)
```

Comment:
Is this still necessary, given [$ prefix changes](https://github.com/rtsao/styletron/issues/196)?

Reply:
The change was required as of: https://github.com/fusionjs/fusion-plugin-font-loader-react/commit/07bd54d21efd9a8cdfc2b87125d2844e328f0d96

I suppose the comment is no longer necessary and we can remove that though.